src/components/ListingCard/ListingCard.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { UUID } from 'crypto';
 import { CaseListing } from '../../types/schema';
-import timestampStringToDate from '../../utils/helpers';
+import { timestampStringToDate, parseDate } from '../../utils/helpers';
 import { CardBody, TagRow, CardTag } from './styles';
 import { H2 } from '../../styles/text';
 import COLORS from '../../styles/colors';
```

Comment:
We might want to update to use `@/` here? Or you can move that to the next sprint task!